app/__init__.py

```diff
@@ -30,5 +32,10 @@ def create_app(test_config=None):
     migrate.init_app(app, db)
```

Comment:
Great job! We need to pass our instance of our `app` & the instance of our SQLALchemy `db` to connect the db and migrate to our Flask app